backend/python/app/rest/intake_routes.py

```diff
@@ -0,0 +1,16 @@
+import json
+
+from flask import Blueprint, current_app, jsonify, request
```

Comment:
nit: remove unused imports
```suggestion
from flask import Blueprint
```